BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The user-specified tags associated with the account."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/BatchAccountUpdateProperties",
+          "description": "The properties of the account."
+        }
+      },
+      "description": "Parameters for updating an Azure Batch account."
+    },
+    "BatchAccountListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BatchAccount"
+          },
+          "description": "The collection of Batch accounts returned by the listing operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token."
+        }
+      },
+      "description": "Values returned by the List operation."
+    },
+    "BatchAccountRegenerateKeyParameters": {
+      "properties": {
+        "keyName": {
+          "type": "string",
+          "description": "The type of account key to regenerate.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AccountKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyName"
+      ],
+      "description": "Parameters supplied to the RegenerateKey operation."
+    },
+    "BatchAccountKeys": {
+      "properties": {
+        "accountName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The Batch account name."
+        },
+        "primary": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The primary key associated with the account."
+        },
+        "secondary": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The secondary key associated with the account."
+        }
+      },
+      "description": "A set of Azure Batch account keys."
+    },
+    "ActivateApplicationPackageParameters": {
+      "properties": {
+        "format": {
+          "type": "string",
+          "description": "The format of the application package binary file."
+        }
+      },
+      "required": [
+        "format"
+      ],
+      "description": "Parameters for an activating an application package."
+    },
+    "ApplicationCreateParameters": {
+      "properties": {
+        "allowUpdates": {
+          "type": "boolean",
+          "description": "A value indicating whether packages within the application may be overwritten using the same version string."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name for the application."
+        }
+      },
+      "description": "Parameters for adding an Application."
+    },
+    "Application": {
```

Comment:
line 1284: looks like this is an ARM resource. This should have `id`, `name` & `type` as the `readOnly`. 

Reference: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/254658613#L1045